### PR TITLE
Replace poller with sockets working version

### PR DESCRIPTION
**Pull request by elliothursh**
## Describe your changes

Make liveReload more robust.

### CODE REVIEW

**Review by alexfarrill — APPROVED** [review]

File: .github/workflows/jest.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ jobs:
           - 5432:5432
     env:
       MM_DEV: 1
+      MM_ENV: test
```
Comment (psugihara) [inline .github/workflows/jest.yml:34]: Looks like you went back to using the hardcoded moduleManifest. Still need this?

**Comment by psugihara** [comment]
switched base to view change in full

**Review by psugihara — APPROVED** [review]

**Comment by psugihara** [comment]
`yarn changeset` before merging?

**Comment by psugihara** [comment]
closing for now

**Pull request closed without merging**